### PR TITLE
Add user to a channel

### PR DESCRIPTION
**Pull request by MartinDelille**
I managed to add a user to a channel and was wondering if you where insterested in this documentation addentum.

### CODE REVIEW

File: source/channels.yaml [hunk 1]
```diff
@@ -167,6 +167,38 @@
         '500':
           description: Could not retrieve the channel stats, the user is not a member of the channel or the channel does not belong to the team
 
+  '/teams/{team_id}/channels/{channel_id}/add':
+    get:
```
Comment (enahum) [inline source/channels.yaml]: This should be a `POST` method

**Comment by MartinDelille** [comment]
Hi @enahum,

Happy to help! I just fixed the issue, is it fine for you?

**Comment by jwilander** [comment]
Much appreciated @MartinDelille !

**Comment by MartinDelille** [comment]
You're welcome! Is there some other api endpoint that would lacks?

**Comment by jwilander** [comment]
I'm actually writing up a bunch of help wanted tickets right now but if you want to get a head start you can take a look at the `/users/search` API, that one is lacking documentation

**Pull request merged**